build.gradle.kts

```diff
@@ -41,15 +41,6 @@ val arrowVersion = "0.10.4"
 val kotestVersion = "4.4.1"
 
 dependencies {
-    implementation(kotlin("stdlib-jdk8") as String) {
-        isForce = true
-        isChanging = true
-    }
-    implementation(kotlin("reflect") as String) {
-        isForce = true
-        isChanging = true
-    }
```

Comment:
Have removed this because [according to the documentation](https://kotlinlang.org/docs/gradle-configure-project.html#dependency-on-the-standard-library) a dependency to `stdlib` is added automatically.

Additionally `isForce` was removed by Gradle (and I am not sure if `isChanging = true` was needed in the first place). An replacement seems to be using ["rich versions"](https://docs.gradle.org/current/userguide/rich_versions.html), in particular `strictly`.
The `isForce = true` was originally added by 7bab16902900867b0bd83a652c18c0bbad95e567, maybe it is not needed anymore at the moment and we can just omit it, as done in this pull request.

Reply:
It was added due to a conflict with a dependency originally